.vscode/settings.json

```diff
@@ -0,0 +1,20 @@
+{
+  "task.autoDetect": "off",
+  "python.terminal.activateEnvInCurrentTerminal": true,
+  "python.testing.pytestEnabled": true,
+  "python.testing.pytestArgs": [
+    "src",
+    "--no-cov",
+    "-s"
+  ],
+  "workbench.iconTheme": "bearded-icons",
```

Comment:
Do we want this included?  I'm not sure what it does but it seems odd to include themes with the project.

Reply:
I actually intended to skip that. 
I ran this command so subsequent updates are ignored. 
```sh
git update-index --skip-worktree .vscode/settings.json
```
This way, if you want you can simply remove those from your settings and git will completely ignore any changes you make to that file.